e2e-tests/index.js

```diff
@@ -0,0 +1,109 @@
+const {
+    connect,
+    keyStores: { InMemoryKeyStore },
+    utils: {
+        KeyPair,
+        format: { parseNearAmount }
+    }
+} = require('near-api-js');
+const { parseSeedPhrase, generateSeedPhrase } = require('near-seed-phrase');
+const { webkit } = require('playwright');
+
+const assert = require('assert');
+
+const NETWORK_ID = 'default';
+const BANK_SEED_PHRASE = process.env.BANK_SEED_PHRASE;
+const BANK_ACCOUNT = process.env.BANK_ACCOUNT;
+const TEST_ACCOUNT_SEED_PHRASE = process.env.TEST_ACCOUNT_SEED_PHRASE;
+const HEADLESS = !['no', 'false'].includes(process.env.HEADLESS);
+
+const config = {
+    networkId: NETWORK_ID,
+    nodeUrl: process.env.NODE_URL || 'https://rpc.testnet.near.org',
+    walletUrl: process.env.WALLET_URL || 'https://wallet.testnet.near.org',
+    keyStore: new InMemoryKeyStore()
+};
+
+async function createIncident(accountId, error) {
+    const { api } = require('@pagerduty/pdjs');
+
+    const pd = api({ token: process.env.PAGERDUTY_API_KEY });
+
+    console.log('Creating incident on PagerDuty')
+    await pd.post('/incidents', {
+        data: {
+            incident: {
+                type: "incident",
+                title: "wallet e2e-tests failure",
+                service: {
+                    id: "PJ9TV6C", // wallet
```

Comment:
Could you put in check if there is an active incident for this, so you don't trigger new incident if there is already an active one on pagerduty.

Also on other note we have similar use case for this e2e tests with bridge. So we could potentially reuse this, but too early to tell for now.

Reply:
@mhalambek I think we just need to solve flakiness (so that there aren't false incidents) vs check for whether there is active incident. I'm working to fix the underlying issues.

Reply:
The problem still is that everytime it runs it opens a new incident for the same issue, not flakiness. If there is the same issue running for 30 minutes and the cron job runs every 5 minutes, 6 incidents will be created which is spamming too much. Pagerduty doesn't know that it is the same incident or not. We need to implement a check which lists all the current open incidents and check if this one is present if not trigger.

Reply:
Gonna change this in follow-up, for now just added flag to enable alerts explicitly